backend/package.json

```diff
@@ -9,10 +9,10 @@
   "main": "src/index.js",
   "scripts": {
     "__migrate": "migrate --compiler 'js:@babel/register' --migrations-dir ./src/db/migrations",
-    "prod:migrate": "migrate --migrations-dir ./dist/db/migrations --store ./dist/db/migrate/store.js",
-    "start": "node dist/",
-    "build": "babel src/ -d dist/ --copy-files",
-    "dev": "nodemon --exec babel-node src/ -e js,gql",
+    "prod:migrate": "migrate --migrations-dir ./build/db/migrations --store ./build/db/migrate/store.js",
+    "start": "node build/",
+    "build": "tsc && mkdir -p build/middleware/helpers/email/templates/ && cp -r src/middleware/helpers/email/templates/*.html build/middleware/helpers/email/templates/ && mkdir -p build/middleware/helpers/email/templates/en/ && cp -r src/middleware/helpers/email/templates/en/*.html build/middleware/helpers/email/templates/en/ && mkdir -p build/middleware/helpers/email/templates/de/ && cp -r src/middleware/helpers/email/templates/de/*.html build/middleware/helpers/email/templates/de/ && mkdir -p build/schema/types/ && cp -r src/schema/types/*.gql build/schema/types/ && mkdir -p build/schema/types/enum/ && cp -r src/schema/types/enum/*.gql build/schema/types/enum/ && mkdir -p build/schema/types/scalar/ && cp -r src/schema/types/scalar/*.gql build/schema/types/scalar/ && mkdir -p build/schema/types/type/ && cp -r src/schema/types/type/*.gql build/schema/types/type/",
```

Comment:
Can this be a script to improve readability?

Reply:
yes, can I do this in a separate ticket? I would do it next if thats acceptable